tests/plugins/elementspath/manual/toolbarfocus.md

```diff
@@ -0,0 +1,31 @@
+@bender-ui: collapsed
```

Comment:
Actually is this test needed? It seems the same as `tests/plugins/elementspath/manual/focus.md`.

Reply:
Looks like rebase artifact, I will remove it right away.